pg-extend/src/pg_datum.rs

```diff
@@ -25,13 +25,10 @@ pub struct PgDatum<'mc>(Option<Datum>, PhantomData<NonNull<&'mc PgAllocator>>);
 
 impl<'mc> PgDatum<'mc> {
     /// Returns a new PgDatum wrapper for Datatypes used by Postgres.
-    pub unsafe fn from_raw<B: Into<pg_bool::Bool>>(
+    pub unsafe fn from_raw(
         _memory_context: &'mc PgAllocator,
-        datum: Datum,
-        is_null: B,
+        datum: Option<Datum>,
```

Comment:
This breaks public API.

Reply:
It looks like a pretty straightforward change. Can drop a note into the CHANGELOG...

Reply:
I created a new `from_option()` method instead. The old datum/isnull code path is is still needed by `pg_fdw` for PostgreSQL 10 and likely to be useful when interacting with Postgres anyway.

Any suggestions for a better name for that method?

Reply:
No. I think this is fine. We could add unsafe to the name, but that’s not really necessary since the fn is always marked unsafe.

Reply:
Yeah, I think `unsafe` in the name would be redundant.